### PR TITLE
chore(service): added additional logging

### PR DESCRIPTION
**Pull request by jsam**
closes: #1013

This PR needs to be merged after #1140 due to the need to add additional logging in those refactored jobs.

### CODE REVIEW

File: renku/service/logging.yaml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  console:
+    class: logging.StreamHandler
+    formatter: simple
+    level: DEBUG
+    stream: "ext://sys.stdout"
+loggers:
+  service:
+    handlers:
+      - console
+    level: DEBUG
```
Comment (rokroskar) [inline renku/service/logging.yaml]: can we make the level configurable through an environment variable so we can easily toggle it in a running service?

Comment (jsam) [reply renku/service/logging.yaml]: Right, I have to test this a bit more after rebase, but this should be already in place and this config should be only for default value.

Comment (rokroskar) [reply renku/service/logging.yaml]: are they exposed through the chart?

Comment (jsam) [reply renku/service/logging.yaml]: I just checked and one is and one is not, I'll add them so both have explicitly set default values there. :+1:

**Pull request merged**